src/index.ts

```diff
@@ -11,6 +11,7 @@ import { tmpdir } from "os";
 import * as path from "path";
 const debug = Debug("impftermin:main");
 import { bgRedBright, whiteBright } from "chalk";
+export var enableAutobooking = false;
```

Comment:
The `configuration.ts` file contains all configuration related concerns. Please do not mix this within different files.

If you change it there to boolean then you do not need this export anymore.